examples/two-chip.rs

```diff
@@ -604,7 +551,7 @@ fn main() {
     // ANCHOR: test-circuit
     // The number of rows in our circuit cannot exceed 2^k. Since our example
     // circuit is very small, we can pick a very small value here.
-    let k = 3;
+    let k = 4;
```

Comment:
`cargo run --example two-chip` is currently failing for me (`InstanceTooLarge`).